repository.rdf

```diff
@@ -397,6 +397,17 @@
 
 
 
+    <!-- http://fedora.info/definitions/v4/repository#RepositoryRoot -->
+
+    <owl:Class rdf:about="http://fedora.info/definitions/v4/repository#RepositoryRoot">
+        <rdfs:label xml:lang="en">repository root</rdfs:label>
+        <owl:subClassOf rdf:resource="http://fedora.info/definitions/v4/repository#Thing"/>
+        <owl:disjointWith rdf:resource="http://fedora.info/definitions/v4/repository#Resource"/>
```

Comment:
Only because the root resource is currently _not_ given the type of `fedora:Resource`. But if you don't think that is correct, I can certainly remove that.